### ISSUE TITLE
Can you guys convert this loss to the tensorflow/keras version, please

### ISSUE THREAD

**Issue by MailSuesarn** [kahnchana/opl#7]

**Comment by MailSuesarn** [comment]
Your work is awesome and very interesting but  Can you guys convert this loss to the TensorFlow/Keras version, please

**Comment by kahnchana** [comment]
Thanks a lot, we will try to add this as well.
